distribution/lib/Standard/Base/0.0.0-dev/src/Data/XML.enso

```diff
@@ -300,7 +311,8 @@ type XML_Element
         XML_Error.handle_java_exceptions <|
             XML_Utils.innerXML self.java_element
 
-    ## Gets elements matching a given tag name.
+    ## GROUP Metadata
+       Gets elements matching a given tag name.
```

Comment:
This also seems much like a `select_columns` kind of operation, my intuition brings it into `Selections`. Why would 'selecting elements based on some criteria' be classified as `Metadata`?

Reply:
will check XML again

Reply:
`get_elements_by_tag_name` seems to have stayed in Metadata. Was this intentional? It seems more suitable for Selections than e.g. `attributes`.

Reply:
supposed to be back in default will remove in next fix.